Controller/RoleController.php

```diff
@@ -327,4 +335,21 @@ public function deleteRoleAssignmentAction(Request $request, $roleAssignmentId)
 
         return $this->redirectToRouteAfterFormPost('admin_roleView', ['roleId' => $roleAssignment->role->id]);
     }
+
+    /**
+     * Lets you select a section which will be used as limitation when assigning the role.
+     *
+     * @param Request $request
+     * @param int $roleId Role ID
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function assignRoleBySectionAction(Request $request, $roleId)
```

Comment:
ok. nevermind. depends on how you read it. as i see you could ADD a section limitation to a Role and you could assign A role TO a group of users. With the Limitation suffix looks better anyway. 

i read it as "assing this role to a section of users". thats what's looked weird to me.